workbench/controllers/RestExplorerController.php

```diff
@@ -70,7 +70,13 @@ public function preExecute() {
         $this->url = str_replace(' ', '+', trim($this->url));
 
         if (in_array($this->requestMethod, RestApiClient::getMethodsWithBodies()) && trim($this->requestBody) == "") {
-            throw new WorkbenchHandledException("Must include a Request Body.");
+            if (stripos($this->requestHeaders, "Content-Type: application/json") !== false) {
+                // If nothing's specified in the JSON body, set it to null.
+                $this->requestBody = "null";
```

Comment:
How about setting this to `{}` instead? That's the usual (but not standard) way to indicate nothingness in JSON other than true `null`, which I'm guess won't work here.